channeldb/db.go

```diff
@@ -336,10 +336,10 @@ type DB struct {
 	noRevLogAmtData bool
 }
 
-// Open opens or creates channeldb. Any necessary schemas migrations due
-// to updates will take place as necessary.
-// TODO(bhandras): deprecate this function.
-func Open(dbPath string, modifiers ...OptionModifier) (*DB, error) {
+// OpenBoltDB opens or creates channeldb using the boltDB type and enables
```

Comment:
Actually I think we can even remove this one, I kept it because in `lnwallet/test_utils.go` we use the `dbModifiers` but not for the opening option but rather when we create the final backend (` CreateWithBackend(backend, modifiers...)`) tho I find it useful in case we want to use the db opening options in the future to have the function ready for it ? But also open to remove it.